### PR TITLE
Add the psychologist's locker

### LINKED ISSUE #928: Feature Request: Psychologist's Locker
Psychologist should probably have a personalised locker, either with their uniform & some random pills(?) or some folders or some filler. Mostly just for something to add to their room.

(I've been told the Psychologist isn't on Wizden so I'm assuming this is okay)

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Adding a psychologist locker.

## Why / Balance
Fixes #928 
Requested a long time ago so, here it is. Also it's good for mapping.

## Technical details
YAML changes.

## Media
image
Contents:
image

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None.

**Changelog**
No CL.

### CODE REVIEW

**Comment by dvir001** [comment]
Move textures under DeltaV

**Comment by Radezolid** [comment]
> Move textures under DeltaV

Tried, it didn't load, dunno why.

**Comment by MilonPL** [comment]
That file is hardcoded iirc

**Comment by dvir001** [comment]
> That file is hardcoded iirc

How? just add
```
  - type: Sprite
    sprite: DeltaV/Structures/Storage/closet.rsi
```

eg: https://github.com/new-frontiers-14/frontier-station-14/blob/36f60e012572c44259dcdbb68a8f73e1de8f2649/Resources/Prototypes/_NF/Entities/Structures/Storage/Closets/Lockers/lockers.yml#L4

**Comment by Radezolid** [comment]
I'm big dum dum. Thanks dvir.

**Comment by Radezolid** [comment]
I'll add a base DeltaV locker in another PR while also moving the justice lockers to DeltaV, untouching upstream files.

File: Resources/Prototypes/DeltaV/Entities/Structures/Storage/Closets/Lockers/lockers.yml [hunk 1]
```diff
@@ -23,3 +23,18 @@
...
     access: [["Clerk"]]
+
+- type: entity
+  parent: LockerBaseSecure
+  id: LockerPsychologist
+  name: psychologist's locker
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Storage/closet.rsi
+  - type: Appearance
```
Comment (dvir001) [inline Resources/Prototypes/DeltaV/Entities/Structures/Storage/Closets/Lockers/lockers.yml]: Appearance coming from base

Comment (Radezolid) [reply Resources/Prototypes/DeltaV/Entities/Structures/Storage/Closets/Lockers/lockers.yml]: True, thanks.

File: Resources/Textures/DeltaV/Structures/Storage/closet.rsi/meta.json
Comment (deltanedas) [inline Resources/Textures/DeltaV/Structures/Storage/closet.rsi/meta.json:1]: 4 space indent for json

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Monotheonist** [comment]
Gonna take the opportunity to point out you have the space to add a hole to the P, and make it look similar to the Pilot locker from the other PR, that's my main sprite nitpick with this PR: 
393260805-e850eb27-f657-497f-b562-39d4f9ae36e4

**Comment by Radezolid** [comment]
tbh, i kinda like this one more, also it's different from the pilot one so more difference rather than just a color change

**Comment by Monotheonist** [comment]
Fair enough, I'm just being the nerd that points that out.

**Review by Lyndomen — APPROVED** [review]
Role parity and fleshing out good

**Review by deltanedas — APPROVED** [review]

**Pull request merged**